src/Microsoft.AspNetCore.SignalR.Common/DefaultProtocols.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public static class DefaultProtocols
+    {
+        public static readonly string JSONProtocol = "json";
```

Comment:
Use `Json`. We generally treat it like a "Word" in our code, even though it is technically an acronym ;)

Reply:
I made it all lowercase to match the name property on the JsonHubProtocol https://github.com/aspnet/SignalR/blob/dev/src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/JsonHubProtocol.cs#L31
I can change that too if we want.

Reply:
The only default protocol we have is JSON really. MSGpack is an addon.

Reply:
Okay, I can remove that from here. Was following what the issue laid out 😄